Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivate.h

```diff
@@ -569,6 +569,8 @@ typedef NS_OPTIONS(NSUInteger, WKDisplayCaptureSurfaces) {
 
 - (void)_requestTargetedElementInfo:(_WKTargetedElementRequest *)request completionHandler:(void(^)(NSArray<_WKTargetedElementInfo *> *))completionHandler WK_API_AVAILABLE(macos(15.0), ios(18.0), visionos(2.0));
 
+- (void)_requestAllTargetableElementsInfo:(CGSize)minimumSize completionHandler:(void(^)(NSArray<NSArray<_WKTargetedElementInfo *> *> *))completionHandler WK_API_AVAILABLE(visionos(2.0));
```

Comment:
You will need to provide availability macros for other platforms too, I believe.

Reply:
@whsieh said that since we only need this for visionOS we should avoid making it available for other platforms, do I need to specify this as unavailable on other platforms?

Reply:
Oh, that's probably a good idea. You could also guard this behind a compile-time flag for visionOS.

Reply:
Just a minor naming nit — the `minimumSize` argument feels a tiny bit unclear here. At first, I'd assumed it was a minimum size threshold for _hit-tested targets_, but then realized it was the size of the intervals between hit-testing locations when I read the implementation.

Not immediately obvious how to many this name clearer (without being overly verbose), but perhaps something along the lines of `hitTestInterval`?

Reply:
Marked them as unavailable but didn't compile them out so it would be easier to catch build errors that could be introduced by changing the names of methods used by this new API when developing on other platforms. Also updated the parameter to `hitTestInterval` ✅